charts/pravega-operator/templates/_helpers.tpl

```diff
@@ -9,8 +9,23 @@ Expand the name of the chart.
 {{/*
 Create a default fully qualified app name.
 We truncate at 63 chars because some Kubernetes name fields are limited to this (by the DNS naming spec).
+If release name contains chart name it will be used as a full name.
```

Comment:
Modified this to allow for a `fullnameOverride` as well, keeping the same pattern as other big OSS projects such as etcd: https://github.com/helm/charts/blob/master/incubator/etcd/templates/_helpers.tpl#L9-L25
Without an override by default this will act just like before.